### ISSUE TITLE
buildTemplate.xml is not present

### ISSUE THREAD

**Issue by marceloandrioni** [olmozavala/OWGIS_V2#26]
Hi, just saw the streamline visualization in [http://viewer.coaps.fsu.edu/GlobalHycom/mapviewer](url), amazing work! I have been using ncWMS and ncWMS2 for a while and would really like to try this new kind of visualization. In the instructions it says to edit file buildTemplate.xml, but this file is not present in the source code. Checking the repository I found a reference in .gitignore and nothing else. Is there another place where I can find this file? Sorry if I am making basic questions, but I never built a war from source before.
Thanks.

**Comment by olmozavala** [comment]
Hello Marcelo, I am glad you like the project. To build a war just create a zip file and change the extension from zip to war :). You already have a tomcat and the template working?  If not, you can download the latest version of OWGIS from here: https://www.dropbox.com/s/o1jq9p65ovtyn3u/OWGIS_V2.war?dl=0
What you would have to do is:
1) Uncompress the file with any unzip software.
2) Modify the /layers/TestLayers.xml with any text editor. To add your own layers from your ncWMS2 server.
3) Compress your folder again, and change the extension from zip to war. 
4) Upload to your tomcat and thats it. 

Other option is to edit the /layers/TestLayers.xml directly from your webapps folder in Tomcat. 
Cheers and I hope it helps.

**Comment by olmozavala** [comment]
Ough and I see that I deleted the buildTemplate.xml. I will uploaded again, sorry about that.

**Comment by marceloandrioni** [comment]
Hi Olmo, got the buildTemplate.xml, thank you. I think I am all set now. Right now I am using a computer without tomcat, but tomorrow I will try to build and deploy and see how it goes. Again, thank you very much and please continue this amazing work. Seeing the current using lagrangian transport  really improves our understanding of the circulation, its way better than simple vectors.

**Comment by marceloandrioni** [comment]
Hi again, I managed to build and deploy OWGISV2 using buildTemplate.xml. After following the examples I added a layer from my ncWMS2 in the file ncWMS_Layers.xml.
`<MainLayers BBOX="-98.0,18.09164810180664,-76.400024,31.960647583007812" 
				format="image/gif"  server="http://localhost:8080/ncWMS2/wms"
				 ncWMStwo="true" >  
		<layer Menu="ncwms,mercator" 
			   EN="mercator"
			   name="mercator/U_VELOCITY:V_VELOCITY-mag" 
			   defParticleSpeed=".9"
			   overlaystreamlines="mercator/U_VELOCITY:mercator/V_VELOCITY" />
	</MainLayers>`
The base layer and magnitude layer show just fine, but the loader for the Streamlines stays in 0%. Do 
I need to change something else? I don't know if this can be a problem, but I am behind a proxy.
Thanks.
![owgis_currents](https://user-images.githubusercontent.com/13545615/32668331-678c3e86-c624-11e7-8a54-62c349a6bf16.png)
